tq42/compute.py

```diff
@@ -36,6 +36,9 @@ def __init__(self, hardware: HardwareProto = HardwareProto.HARDWARE_UNSPECIFIED)
             gpu=configuration.get("GPU", ""),
         )
 
+    def __repr__(self):
+        return f"(Hardware name: { self.config.name}, Cpu: { self.config.cpu}, Gpu: { self.config.gpu}, Memory: { self.config.memory}, Storage: { self.config.storage})"
```

Comment:
Can we please do some proper formatting for the variables 🙈 These spaces are freaking me out a bit 😄

Reply:
done it using dynamic data trasformation approach